src/geo/transform.js

```diff
@@ -146,7 +171,7 @@ class Transform {
      *
      * @private
      */
-    getVisibleWrappedCoordinates(tileCoord) {
+    getVisibleWrappedCoordinates(tileCoord: {x: number, y: number, z: number}) {
```

Comment:
It depends. In this case, I think it's OK since we don't actually require any methods or other properties of the TileCoord class. If we ever do, flow will complain and the type can be refined.